### ISSUE TITLE
Please release the next version. :-)

### ISSUE THREAD

**Issue by pnathan** [clj-commons/iapetos#9]
Hi,

I'd like to be able to use the custom bucketing for my services.

Best regards,
pnathan

**Comment by xsc** [comment]
@pnathan I don't think there have been any changes to iapetos since the last release (0.1.6). The fix for custom bucketing in the Ring collector (see #5) has already been included since 0.1.5.

Or are you referring to something else?

**Comment by pnathan** [comment]
PEBKAC.

careful study of code resulted in this diff:
```
-       :latency-histogram-buckets histogram-buckets-by-seconds)))
+        {:latency-histogram-buckets histogram-buckets-by-seconds})))
```
